editor/src/app/sites/sites-state/sites.state.ts

```
@@ -323,4 +324,15 @@ export class SitesState implements NgxsOnInit {
     // Current workaround is to reload window to get correct state
     return this.appStateService.sync('siteThemeApply', action.payload, 'PUT');
   }
+
+  @Action(SwitchContentsSitesAction)
+  SwitchContentsSites({}, action: SwitchContentsSitesAction) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)